### DISCUSSION TITLE
Api V2 returns 404

### DISCUSSION THREAD

**Discussion by arminus** [nocodb/nocodb#6688] [Q&A] [answered]
Quick question: It says here https://github.com/nocodb/nocodb/releases that there is an Api V2 now, however, on my self-hosted latest docker image I get 404s for anything in v2, e.g. http://localhost:8080/api/v2/meta/projects/

What am I missing?

**Comment by pranavxc** [comment, marked as answer]
Sorry for the confusion, we had renamed `project` to `base` and `base`(datasource) to `source`, so the new dat apis is `http://localhost:8080/api/v2/meta/bases/`. 

You can find the new meta api documentation here - https://meta-apis-v2.nocodb.com

**Reply by arminus** [reply]
OK, so the docs hadn't been updated, thanks. But Swagger is also still on v1...?
